### ISSUE TITLE
Errors in server create cmd is not handled

### ISSUE THREAD

**Issue by nvthongswansea** [gridscale/gscloud#88]
The error in this line [L159](https://github.com/gridscale/gscloud/blob/master/cmd/server.go#L159) should be handled. It is better if the command can be rollbacked when there is an error returned. E.g: if linking a server and a storage fails, the server and the storage should be removed.

**Comment by bkircher** [comment]
> E.g. if linking a server and a storage fails, the server and the storage should be removed.

Yep. Actually if you take a look closely at how the panel does this, you see POST /batch that will do exactly this.
